man/waybar-sway-workspaces.5.scd

```diff
@@ -65,10 +65,6 @@ Addressed by *sway/workspaces*
     default: empty ++
     Lists workspaces that should always be shown, even when non existent
 
-*on-update*: ++
```

Comment:
Since sway/workspaces is not a Label module,  it has no common config, should we keep it here?

Reply:
Updated to keep this config.